skiko/src/commonTest/kotlin/org/jetbrains/skia/ParagraphTest.kt

```diff
@@ -172,4 +179,35 @@ class ParagraphTest {
             }
         }
     }
+
+    @Test
+    fun `layout paragraph with its maxIntrinsicWidth shouldn't lead to wraps`() = runTest {
```

Comment:
Web CI pipeline failed because k/js and k/wasm don't like the test names in `...`.

So changing the name should help: 
```suggestion
    fun layout_paragraph_with_its_maxIntrinsicWidth_shouldnt_lead_to_wraps() = runTest {
```